<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Feedback</class>
 <widget class="QDialog" name="Feedback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>193</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblInfo">
       <property name="text">
        <string>0/250</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Feedback</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
